doc/init/init-bulk.md

```diff
@@ -0,0 +1,79 @@
+
+The following table gives explicit descriptions on keys in `PARAM`.
+
+The bold notation of key (such as **Elements**) means that it's a necessary key.
+
+ Key  | Type          | Example                                                      | Description                                                      |
+| :---------------- | :--------------------- | :-------------------------------------- | :-------------------------------------------------------------|
+| **stages** | List of Integer | [1,2,3,4] | Stages for `init_bulk`
+| **Elements** | List of String | ["Mg"] | Atom types
+|  cell_type | String  | "hcp" | Specifying which typical structure to be generated. **Options** include fcc, hcp, bcc, sc, diamond.
+| latt | Float | 4.479 | Lattice constant for single cell.
+| from_poscar | Boolean | True | Deciding whether to use a given poscar as the beginning of relaxation. If it's true, keys (`cell_type`, `latt`) will be aborted. Otherwise, these two keys are **necessary**.
+| from_poscar_path | String | "....../C_mp-47_conventional.POSCAR" | Path of POSCAR for VASP or STRU for ABACUS. **Necessary** if `from_poscar` is true.
+| relax_incar | String | "....../INCAR" | Path of INCAR for VASP or INPUT for ABACUS for relaxation in VASP. **Necessary** if `stages` include 1.
+| md_incar | String |  "....../INCAR_md" | Path of INCAR for VASP or INPUT for ABACUS for MD in VASP. **Necessary** if `stages` include 3.|
+| **scale** | List of float | [0.980, 1.000, 1.020] | Scales for transforming cells.
+| **skip_relax** | Boolean | False | If it's true, you may directly run stage 2 (perturb and scale) using an unrelaxed POSCAR.
+| **pert_numb** | Integer | 30 | Number of perturbations for each POSCAR.
+| **pert_box** | Float | 0.03 | Percentage of Perturbation for cells.
+| **pert_atom** | Float | 0.01 | Perturbation of each atoms (Angstrom).
+| **md_nstep** | Integer | 10 | Steps of AIMD in stage 3. If it's not equal to settings via `NSW` in `md_incar`, DP-GEN will follow `NSW`.
+| **coll_ndata** | Integer | 5000 | Maximal number of collected data.
+| type_map | List | [ "Mg", "Al"] | The indices of elements in deepmd formats will be set in this order.
+| init_fp_style | String | "ABACUS" or "VASP" | First-principle software. If this key is absent, the default value will be "VASP".
+| relax_kpt | String | "....../KPT" | Path of `KPT` file for relaxation in stage 1. Only useful if `init_fp_style` is "ABACUS".
+| md_kpt | String | "....../KPT" | Path of `KPT` file for MD simulations in stage 3. Only useful if `init_fp_style` is "ABACUS".
+| atom_masses | List of float | [24] | List of atomic masses of elements. The order should be the same as `Elements`. Only useful if `init_fp_style` is "ABACUS".
```

Comment:
This table should be generated automatically by `dargs`. But I don't request it done by this PR.
cc @HuangJiameng